pkg/grafana/datasource-handler.go

```diff
@@ -26,29 +26,33 @@ func (h *DatasourceHandler) GetFullName() string {
 	return "grafana.datasource"
 }
 
-// GetJSONPath returns a paths within Jsonnet output that this provider will consume
-func (h *DatasourceHandler) GetJSONPath() string {
-	return "grafanaDatasources"
+const datasourcesPath = "grafanaDatasources"
+
+// GetJSONPaths returns paths within Jsonnet output that this provider will consume
+func (h *DatasourceHandler) GetJSONPaths() []string {
+	return []string{
+		datasourcesPath,
+	}
 }
 
 // GetExtension returns the file name extension for a datasource
 func (h *DatasourceHandler) GetExtension() string {
 	return "json"
 }
 
-func (h *DatasourceHandler) newDatasourceResource(uid, filename string, source Datasource) grizzly.Resource {
+func (h *DatasourceHandler) newDatasourceResource(path, uid, filename string, source Datasource) grizzly.Resource {
 	resource := grizzly.Resource{
 		UID:      uid,
 		Filename: filename,
 		Handler:  h,
 		Detail:   source,
-		Path:     h.GetJSONPath(),
+		Path:     path,
```

Comment:
What kind of path is this? Filesystem? Json? URL? Can you make it more succinct?

Reply:
yah, fair enough, before this change it was clear it was a jsonPath, but isn't now.